README.md

```diff
@@ -1 +1,5 @@
-# tic-tac-toe
+# tik-tac-toe
```

Comment:
you misspelled tic-tic-toe

Reply:
Okay ill fix that real quick

Reply:
https://github.com/thawki/tic-tac-toe/pull/1/commits/7f0ccc2e54bc702e0b16965af0d2f0f682a85a82 resolves spelling error